tools/install.sh

```diff
@@ -113,28 +115,33 @@ function _omb_install_run {
 function _omb_install_banner {
   # MOTD message :)
   printf '%s' "$GREEN"
-  printf '%s\n' \
-    '         __                          __               __  ' \
-    '  ____  / /_     ____ ___  __  __   / /_  ____ ______/ /_ ' \
-    ' / __ \/ __ \   / __ `__ \/ / / /  / __ \/ __ `/ ___/ __ \' \
-    '/ /_/ / / / /  / / / / / / /_/ /  / /_/ / /_/ (__  ) / / /' \
-    '\____/_/ /_/  /_/ /_/ /_/\__, /  /_.___/\__,_/____/_/ /_/ ' \
-    '                        /____/                            .... is now installed!'
+  local BANNER
+  BANNER=$(cat <<-'EOF'
+	         __                          __               __  
+	  ____  / /_     ____ ___  __  __   / /_  ____ ______/ /_ 
+	 / __ \/ __ \   / __ `__ \/ / / /  / __ \/ __ `/ ___/ __ \
+	/ /_/ / / / /  / / / / / / /_/ /  / /_/ / /_/ (__  ) / / /
+	\____/_/ /_/  /_/ /_/ /_/\__, /  /_.___/\__,_/____/_/ /_/ 
+	                        /____/                            .... is now installed!
+EOF
+  )
+  printf '%s\n' "$BANNER"
   printf '%s' "$NORMAL"
 }
 
 ## @fn _omb_install_user_bashrc
 ##   @var[in] install_opts
 ##   @var[in] OSH
 function _omb_install_user_bashrc {
-  printf "${BLUE}Looking for an existing bash config...${NORMAL}\n"
+  printf '%s\n' "${BLUE}Looking for an existing bash config...${NORMAL}"
   if [[ -f ~/.bashrc || -h ~/.bashrc ]]; then
-    local bashrc_backup=~/.bashrc.omb-backup-$(date +%Y%m%d%H%M%S)
-    printf "${YELLOW}Found ~/.bashrc.${NORMAL} ${GREEN}Backing up to $bashrc_backup${NORMAL}\n"
+    local bashrc_backup
+    bashrc_backup=~/.bashrc.omb-backup-$(date +%Y%m%d%H%M%S)
```

Comment:
Here again we do not use the exit status, so there is no reason to put them in separate lines.

Reply:
Thanks for catching that. I've added an ignore line for this hint.